src/messages/security_metadata.rs

```diff
@@ -16,31 +16,7 @@ use crate::{
 };
 use bincode::serialize;
 use serde::{Deserialize, Serialize};
-use std::{
-    collections::BTreeSet,
-    fmt::{self, Debug, Formatter},
-};
-
-/// Metadata needed for verification of the sender.
-/// Contain shares of the section signature before combining into a BLS signature
-/// and into a FullSecurityMetadata.
-#[derive(Eq, PartialEq, Clone, Hash, Serialize, Deserialize)]
-pub struct PartialSecurityMetadata {
```

Comment:
It probably make more sense to fold that commit with the previous one since we are adding AccumulatingMessage to functionally replace PartialSecurityMetadata, and that removal change is pretty small.